### PR TITLE
MySQL Task Dmitrii.Orlenko

### PR DESCRIPTION
**Pull request by enerGOww**
https://travis-ci.org/github/enerGOww/db-assignments/builds/711892263

### CODE REVIEW

File: task/sql-tasks.js [hunk 1]
```diff
@@ -178,7 +281,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            COUNT(*) AS 'TotalOfCurrentProducts',
+            SUM(Discontinued) AS 'TotalOfDiscontinuedProducts'
```
Comment (AlexMokin) [inline task/sql-tasks.js]: that works, but it's better to use count explicitly if you need to count records.

Comment (AlexMokin) [reply task/sql-tasks.js]: but in this case it's better to just use 3 sql statements, on without a from-clause
e.g.
```
select 
   (select count(*) from ....) as 'T'
   (select count(*) from ....) as 'TT'
```

File: task/sql-tasks.js [hunk 2]
```diff
@@ -200,7 +317,24 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            SUM(MONTH(OrderDate) = 1) AS 'January',
```
Comment (AlexMokin) [inline task/sql-tasks.js]: use count when you need to count records. It shows intent better

**Review by AlexMokin — COMMENTED** [review]
Looks good. See comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by enerGOww** [comment]
https://travis-ci.org/github/enerGOww/db-assignments/builds/721594743  done it

**Comment by AlexMokin** [comment]
Accepted

**Pull request closed without merging**